apps/native-component-list/screens/TextToSpeechScreen.js

```diff
@@ -126,6 +146,11 @@ export default class TextToSpeechScreen extends React.Component {
     });
   };
 
+  _showAllVoices = async () => {
+    let availableVoices = await Speech.getAvailableVoicesAsync();
+    console.log('availableVoices ', JSON.stringify(availableVoices));
```

Comment:
NCL is not only used in debug mode, we need to show this to the user 😉 
```suggestion
   alert('Available voices: ' + availableVoices);
```